vms/evm/subscriber.go

```diff
@@ -116,25 +228,15 @@ func (s *subscriber) Subscribe() error {
 }
 
 func (s *subscriber) dialAndSubscribe() error {
-	// Dial the configured destination chain endpoint
+	// Dial the configured source chain endpoint
 	// This needs to be a websocket
-	ethClient, err := ethclient.Dial(s.nodeURL)
+	ethClient, err := ethclient.Dial(s.nodeWSURL)
 	if err != nil {
 		return err
 	}
 
-	filterQuery := interfaces.FilterQuery{
-		Topics: [][]common.Hash{
-			{warp.WarpABI.Events["SendWarpMessage"].ID},
-			{},
-			{},
-		},
-		Addresses: []common.Address{
-			warp.ContractAddress,
-		},
-	}
+	filterQuery := warpFilterQuery
```

Comment:
Is having this line functionally different than passing `warpFilterQuery` directly into `ethClient.SubscribeFilterLogs`?

Reply:
Nope, I think that was leftover from an earlier iteration. Cleaned up now.